### DISCUSSION TITLE
Question about Volumtric Fraction

### DISCUSSION THREAD

**Discussion by Calvinsgithub** [OpenRadioss/OpenRadioss#2810] [Q&A]
Dear @MarianBulla ,
I hope you are doing well.

Thank you for developing the excellent software "OpenRadioss.".I'm new to this app, and I have a question for you. Recently, I simulated metal cutting using the cel method under the Radioss plate. I used type 18 contact type and material with LAW51, and the metal EOS option was POLYNOMIAL.Volumtric Fraction is shown in the Hyperview calculation results.Why does the material volume behind the tool enter the upper air grid area 1 after being cut by the tool? (The blue part in the picture represents the metal material Alu, and the red part represents the air.)？.I added the model file. Due to insufficient familiarity with the software, I hope to get your guidance.

![`0DUD4XYUO`8V6M0`%P(9N9](https://github.com/user-attachments/assets/c777472e-3afb-4f0b-94b2-592028d01693)

[cel_cutting.zip](https://github.com/user-attachments/files/16231804/cel_cutting.zip)

Best regards,
Calvins

**Comment by RevolT87** [comment]
Hi @Calvinsgithub,
Thanks for sharing your model. 
Your model is quite correct except for 1 thing  you can improve that I will explain later in this message.
Regarding your question, the results you see are as expected, IMO. You would need a much stiffer interface to push the air, but it wouldn't be suitable in term of result quality & time step. You should ignore the air that is inside the steel, as it has almost no stiffness. Moreover, you cannot have void in LAW51 to replace the air. 

Here is a picture of the pressure field along with an isovalue of the aluminium. We can see that the behavior is globaly correct with a coarse mesh:
![image](https://github.com/user-attachments/assets/da55deb3-d546-4250-9f95-1dc07ffa96b8)


Concerning the things you can inprove:
the initial pressure is not well balanced: this is due to the EoS that gives a pressure of 0.1 for the air and of 0 for aluminium. You need to correct one or the other to have either a 0 pressure or a 0.1 pressure initially depending wether you want to work with absolute or relative pressure.
![image](https://github.com/user-attachments/assets/5ed65539-9eb2-4b87-bc48-6f8db8f5be78)

I hope this help. 
Regard,
Mathis.

**Comment by elequiniou** [comment]
[like]  Eric Lequiniou reacted to your message:
________________________________
From: Mathis Loverini ***@***.***>
Sent: Monday, July 15, 2024 4:27:16 PM
To: OpenRadioss/OpenRadioss ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] Re: [OpenRadioss/OpenRadioss] Question about Volumtric Fraction (Discussion #2810)


CAUTION: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

Hi @Calvinsgithub<https://github.com/Calvinsgithub>,
Thanks for sharing your model.
Your model is quite correct except for 1 thing you can improve that I will explain later in this message.
Regarding your question, the results you see are as expected, IMO. You would need a much stiffer interface to push the air, but it wouldn't be suitable in term of result quality & time step. You should ignore the air that is inside the steel, as it has almost no stiffness. Moreover, you cannot have void in LAW51 to replace the air.

Here is a picture of the pressure field along with an isovalue of the aluminium. We can see that the behavior is globaly correct with a coarse mesh:
image.png (view on web)<https://github.com/user-attachments/assets/da55deb3-d546-4250-9f95-1dc07ffa96b8>

Concerning the things you can inprove:
the initial pressure is not well balanced: this is due to the EoS that gives a pressure of 0.1 for the air and of 0 for aluminium. You need to correct one or the other to have either a 0 pressure or a 0.1 pressure initially depending wether you want to work with absolute or relative pressure.
image.png (view on web)<https://github.com/user-attachments/assets/5ed65539-9eb2-4b87-bc48-6f8db8f5be78>

I hope this help.
Regard,
Mathis.
